scripts/setup.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=${DIR}/..
+
+setup_node_env() {
+  echo "Attempting to switch node versions for you..."
+  if command -v fnm &> /dev/null
+  then
+      fnm use
+  else
+    if command -v nvm &> /dev/null
+    then
+      echo "You are using 'nvm', 'fnm' is preferred (it's quicker)!"
+      nvm use
+    else
+      echo "Please install fnm: 'brew install fnm'"
+      exit 1
+    fi
+  fi
+}
+
+check_node_version() {
+  runningNodeVersion=$(node -v)
+  requiredNodeVersion=$(cat "$ROOT_DIR/.nvmrc")
+
+  # remove leading v
+  runningNodeVersionNumber=${runningNodeVersion//[v]/}
+
+  if [ "$runningNodeVersionNumber" != "$requiredNodeVersion" ]; then
+    echo -e "Using wrong version of Node. Required ${requiredNodeVersion}. Running ${runningNodeVersion}."
+    setup_node_env
+  fi
+}
+
+install_node_deps() {
+  echo "Installing node dependencies"
+  if command -v yarn &> /dev/null
+  then
+    yarn install
+  else
+    echo "yarn not found, installing..."
+    npm install --global yarn
+    yarn install
+  fi
+}
+
+print_instructions() {
+  cat <<- "EOF"
+------------------------
+You're all set up :)
+
+To run a project, from the root directory:
+
+# yarn {appName = content|catalogue|identity}
```

Comment:
I think there's only terraform things left in catalogue, [content and catalogue were merged](https://github.com/wellcomecollection/wellcomecollection.org/pull/9991) as they were more and more mixed with the coming of the search hub. So `yarn content` runs both and `yarn catalogue` was removed.